travis-ci.sh

```diff
@@ -25,3 +25,9 @@ if [ ${RUN_TEST=1} -eq 1 ]; then
         dub --temp-build --compiler=$DC --single $ex # --override-config vibe-core/$CONFIG
     done
 fi
+
+# test compatibility with current Vibe.d
+dub add-local .
+git clone https://github.com/rejectedsoftware/vibe.d
```

Comment:
`--depth 1` makes sense here to reduce the network traffic a bit (mainly skips the old Windows library binaries for OpenSSL and libevent).